src/test/java/guitests/InvalidLoginTest.java

```diff
@@ -17,9 +17,8 @@ public void launchApp() {
 
     @Test
     public void invalidLoginTest() throws InterruptedException {
-        TextField repoOwnerField = find("#repoOwnerField");
-        doubleClick(repoOwnerField);
-        doubleClick(repoOwnerField);
+        find("#repoOwnerField");
```

Comment:
The `find` calls shouldn't be needed any more. Unless we want to click in those fields to make sure they're in focus?